types.d.ts

```diff
@@ -0,0 +1,294 @@
+     * srcPath points to a local file set `options.localSrc` to true
+     * @param {module:types~RemotePathString} destPath copy destination path to a file or directory. If
+     * destPath points to a local file set `options.localDest` to true
+     * @param {object} [options={}] remoteCopyOptions
+     * @param {boolean} [options.localSrc = false] Set this option to true to copy
+     * files from the local file system. Cannot be combined with localDest.
+     * @param {boolean} [options.localDest = false] Set this option to true to
+     * copy files to the local file system. Cannot be combined with localSrc.
+     * @param {boolean} [options.noOverwrite = false] set to true to overwrite
+     * existing files
+     * @param {Function} [options.progressCallback] a function that will be called
+     * every time the operation completes on a single file,the srcPath and destPath to the copied file
+     * are passed as argument to the callback `progressCallback(srcPath, destPath)`
+     * @returns {Promise<object<string, string>>} returns a promise resolving to an object containing all copied files
+     * from src to dest `{ srcFilePath: destFilePath }`
+     * @memberof Files
+     */
+    copy(srcPath: module, destPath: module, options?: {
+        localSrc?: boolean;
+        localDest?: boolean;
+        noOverwrite?: boolean;
+        progressCallback?: (...params: any[]) => any;
+    }): Promise<{
+        [key: string]: string;
+    }>;
+}
+
+/**
+ * Initializes and returns the cloud files SDK.
+ *
+ * To use the SDK you must either provide provide your
+ * [OpenWhisk credentials]{@link module:types~OpenWhiskCredentials} in
+ * `credentials.ow` or your own
+ * [Azure blob storage credentials]{@link module:types~AzureCredentialsAccount} in `credentials.azure`.
+ *
+ * OpenWhisk credentials can also be read from environment variables (`__OW_NAMESPACE` and `__OW_API_KEY`).
+ *
+ * @param {object} [config={}] configuration used to init the sdk
+ *
+ * @param {module:types~OpenWhiskCredentials} [config.ow]
+ * {@link module:types~OpenWhiskCredentials}. Set those if you want
+ * to use ootb credentials to access the state management service. OpenWhisk
+ * namespace and auth can also be passed through environment variables:
+ * `__OW_NAMESPACE` and `__OW_API_KEY`
+ *
+ * @param {module:types~AzureCredentialsAccount|module:types~AzureCredentialsSAS} [config.azure]
+ * bring your own [Azure SAS credentials]{@link module:types~AzureCredentialsSAS} or
+ * [Azure storage account credentials]{@link module:types~AzureCredentialsAccount}
+ *
+ * @param {object} [config.tvm] tvm configuration, applies only when passing OpenWhisk credentials
+ * @param {string} [config.tvm.apiUrl] alternative tvm api url.
+ * @param {string} [config.tvm.cacheFile] alternative tvm cache file, set to `false` to disable caching of temporary credentials.
+ * @returns {Promise<Files>} A Files instance
+ */
+declare function init(config?: {
+    ow?: module;
```

Comment:
instead of module it should be OpenWhiskCredentials here. Not sure how to annotate the JSDoc properly